backend/config/__init__.py

```diff
@@ -29,6 +29,8 @@
 REDIS_HOST: Final = os.environ.get("REDIS_HOST", "127.0.0.1")
 REDIS_PORT: Final = os.environ.get("REDIS_PORT", 6379)
 REDIS_PASSWORD: Final = os.environ.get("REDIS_PASSWORD")
+REDIS_USERNAME: Final = os.environ.get("REDIS_USERNAME")
```

Comment:
This needs a fallback since it's not required

Reply:
Was falling back to None to match default values of `redis.Redis`. Further inverstigation reveales Redis handles `None` and `""` as same.
So, falling back to empty string would be better as this also cleans up `redis_url` definition.